src/EnergyPlus/DaylightingManager.cc

```diff
@@ -6114,7 +5980,245 @@ void DayltgHitBetWinObstruction(EnergyPlusData &state,
     }
 }
 
-void DayltgInteriorIllum(EnergyPlusData &state, int const daylightCtrlNum) // Daylighting:Controls number
+void initDaylighting(EnergyPlusData &state, bool const initSurfaceHeatBalancefirstTime)
```

Comment:
Two new functions for daylighting code that was sitting over in the middle of `HeatBalanceSurfaceManager::InitSurfaceHeatBalance`. Copied verbatim from there in 2 parts here. It's possible some of this could be skipped if there's no daylighting, but it wasn't being skipped before, so do no harm here.